### PR TITLE
Hivemind-ery Update

### PR DESCRIPTION
**Pull request by Wouju**
## About The Pull Request

Updates hivemind-convertable machinery, adds one extra surname, reduces the time it takes for hivemind to be announced and fixes hivemind death not being announced. Makes some of the code easier to look at.

## Why It's Good For The Game

Disposal bins aren't grouped up together in a single room in large amounts (unlike plumbing ducts), this allows the hivemind to have one extra convertable machine per room.
Extra surname is more of a cosmetic change, also vidya game reference.
Instead of taking 300 (seconds?) it takes 180 seconds for hivemind to be announced, that's 3 minutes, enough for hivemind to spread for a fair bit if not discovered earlier.
Hivemind death was meant to be announced earlier, somehow I fucked that up earlier in my Hivemind Rebalance PR.

## Changelog
:cl:
add: Added new hivemind surname, added ability for hivemind to convert disposal bins
del: Removed hivemind ability to convert plumbing ducts
fix: Hivemind death is now properly announced
refactor: code is easier to look at, i hope (it looks more like an actual list and you don't have to scroll to the side too much)
/:cl:

### CODE REVIEW

**Review by genessee596 — APPROVED** [review]
it is time
Still needs maintainer approval.

**Review by TorinoFermic — APPROVED** [review]

**Comment by genessee596** [comment]
Delegate and maintainer approval. Merging now.

**Pull request merged**
